doc/makeflow.html

```diff
@@ -212,6 +212,12 @@ <h3 id="workqueue.projectnames">Project Names<a class="sectionlink" href="#workq
 
 <code>work_queue_worker -N proj1 -N proj2 -N proj3</code>
 
+<p>
+In addition to creating a project name using the -N option, this will also trigger automatic reporting to the 
+designated catalog server. The Port and Server address are taken from the environment variables <b>CATALOG_HOST</b> 
+and <b>CATALOG_PORT</b>. If either variable is not set, then the addresses "catalog.cse.nd.edu,backup-catalog.cse.nd.edu" and port 9097 will be used.
+</p>
+
```

Comment:
Suggest that you clearly separate two things in the docs:
- Indicate earlier on that -N can be used with any batch system type.
- Indicate here that -N is the easiest way to match workers to masters.